securedrop/db.py

```diff
@@ -132,7 +132,8 @@ class Submission(Base):
         "Source",
         backref=backref(
             'submissions',
-            order_by=id))
+            order_by=id,
+        cascade="delete"))
```

Comment:
Nit: formatting. Either put all of the parameter to `backref` on one line, or left-align them. If you can fit them all of one line, that is preferable.